rust/operator-binary/src/controller/build.rs

```diff
@@ -0,0 +1,470 @@
+mod config;
```

Comment:
I always thought [build.rs](https://doc.rust-lang.org/cargo/reference/build-scripts.html) is a bit special for cargo.

Reply:
Only in the root of a package.